client/src/components/App/index.js

```diff
@@ -1,11 +1,12 @@
 import React, { Component } from 'react';
+import Nav from '../Nav';
 import './App.css';
 
 class App extends Component {
   render() {
     return (
       <div>
-          Josh
+          <Nav />
```

Comment:
Question: React automatically replaces custom named tags with the results of `render()` of a class of the same name?